packages/terra-avatar/src/variants/Generic.jsx

```diff
@@ -2,12 +2,16 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import styles from '../common/Avatar.module.scss';
-import {
-  AVATAR_VARIANTS, setColor,
-} from '../common/AvatarUtils';
+import { setColor } from '../common/AvatarUtils';
 
 const cx = classNames.bind(styles);
 
+const GENERIC_VARIANTS = {
```

Comment:
@supreethmr we will have to export this separately so that consumer can access it.

Reply:
Added export : 4a470cf434de88c60b8cea36e209847b6951100c